tests/Tests/200__monitor_and_manage/200__metrics/203__alerts.robot

```diff
@@ -47,9 +68,134 @@ Verify Alert RHODS-PVC-Usage-At-100 Is Fired When User PVC Is At 100 Percent
     ...    ${RHODS_PROMETHEUS_TOKEN}
     ...    RHODS-PVC-Usage
     ...    User notebook pvc usage at 100%
+    ...    alert-duration=120
 
     [Teardown]    Teardown PVC Alert Test
 
+Verify Alert "RHODS Route Error Burn Rate" Is Fired When Traefik Is Down    # robocop: disable:too-long-test-case
+    [Documentation]    Verifies that alert "RHODS Route Error Burn Rate" is fired when traefik-proxy is not working
+    [Tags]    Tier3    ODS-738
+
+    Skip    msg=This alert was disabled in RHODS 1.3.0. More info at RHODS-2101
+
+    Skip Test If Alert Is Already Firing    ${RHODS_PROMETHEUS_URL}
+    ...    ${RHODS_PROMETHEUS_TOKEN}
+    ...    SLOs-haproxy_backend_http_responses_total
+    ...    RHODS Route Error Burn Rate
+    ...    alert-duration=120
+
+    ODS.Scale Deployment    redhat-ods-operator    rhods-operator    replicas=0
+    ODS.Scale Deployment    redhat-ods-applications    traefik-proxy    replicas=0
+
+    Prometheus.Wait Until Alert Is Firing    ${RHODS_PROMETHEUS_URL}
+    ...    ${RHODS_PROMETHEUS_TOKEN}
+    ...    SLOs-haproxy_backend_http_responses_total
```

Comment:
I'd explicitly write the argument names for readability purposes

Reply:
I've added it in some parameters where there could be confusion (like alert-duration and timeout) but I prefer not to add it to all of them, at least not in this PR